### DISCUSSION TITLE
Add note for each recognition

### DISCUSSION THREAD

**Discussion by ltguillaume** [aleksey-saenko/MusicRecognizer#36] [Ideas]
Oftentimes it's really useful to add a note to a recognition:
- Add the movie/trailer/video/etc. the song was in
- Add the club/bar/cafe/etc. the song was played in
- Add who recommended the song

So it would be nice to have a Note/Notes field for every recognition to add some custom info.

**Comment by aleksey-saenko** [comment]
I've seen something like this before, it seems to be implemented quite well in SoundHound app. Although it's not the primary feature for a music recognition app, it can be useful. For a good implementation, it will also be necessary to have some search functionality for tags and notes, as well as filters for the library.

However, I can't say that I will take on this task in the near future. I find the current architecture of the app's code is inconvenient, I would like to work on this issue. I also have doubts about the user interface, the dependency on specific recognition service, and other things. I can't say when I will work on the notes. But thank you for the suggestion, I have added it to the backlog.

**Reply by ltguillaume** [reply]
Although these search/organization features you mention might all be very nice to have, I agree that those are not really within the scope of such an app. I do think, however, that it's rather vital to have some context about why I chose to do this recognition and in what situation that was. Otherwise, the recognition itself loses its value.

**Comment by ltguillaume** [comment]
@aleksey-saenko 
Would you please reconsider adding a simple notes field? I so dearly miss this.

**Reply by aleksey-saenko** [reply]
I still believe this feature should be added — the only question is how exactly it should be implemented.
If we have an open issue #103 , let’s continue there.
